README.md

```diff
@@ -4,75 +4,44 @@ Servo is a prototype web browser engine written in the
 [Rust](https://github.com/rust-lang/rust) language. It is currently developed on
 64-bit macOS, 64-bit Linux, 64-bit Windows, and Android.
 
-Servo welcomes contribution from everyone.  See
-[`CONTRIBUTING.md`](CONTRIBUTING.md) and [`HACKING_QUICKSTART.md`](docs/HACKING_QUICKSTART.md)
-for help getting started.
+Servo welcomes contribution from everyone. Check out [The Servo Book](https://book.servo.org) to get started, or go to [servo.org](https://servo.org/) for news and guides.
 
-Visit the [Servo Project page](https://servo.org/) for news and guides.
+## Quickstart
```

Comment:
Let's call this "Getting started" to avoid jargon:

```suggestion
## Getting started
```

Reply:
done in bf938ae761742a4a7ecdf697cbbda17ebc059e9c